content/metadata.yaml

```
@@ -465,3 +465,22 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2020-04-30
+-
```

```suggestion
  -
```
noticed one more thing